docs/release-notes/8.2.0-1.md

```diff
@@ -0,0 +1,44 @@
+# Percona XtraBackup 8.2.0-1 (2023-11-)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//quickstart-overview.md).
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+This is an Innovation release. This type of release is only supported for a short time and is designed to be used in an environment with fast upgrade cycles. Developers and DBAs are exposed to the latest features and improvements. Patches and security fixes are included in the next Innovation release instead of a patch release or fix release within an Innovation release. To keep your environment current with the latest patches or security fixes, upgrade to the latest release.
+
+## Release highlights
+
+Percona XtraBackup 8.2.0-1 is based on MySQL 8.2 and fully supports the Percona Server for MySQL 8.2 Innovation series and the MySQL 8.2 Innovation series. This release allows taking backups of Percona Server 8.2.0-1 and MySQL 8.2.
+
+Use the Percona XtraBackup 8.0 series to take backups of Percona Server for MySQL 8.0.x or MySQL 8.0.x. Percona XtraBackup 8.2.0-1 does not take a backup of the Percona Server for MySQL 8.0 or the MySQL 8.0.x series.
+
+## Bug fixes
+
+* [PXB-3003]: Percona XtraBackup discovering redo logs to parse and the server purging redo logs at the same time could cause a race condition.
+
+* [PXB-3171]: Due to rework, the xbcloud binary had lost the ability to check the return code of the `diff` command, which ensures we only download required files in a partial download.
+
+* [PXB-3174] : XtraBackup failed to execute the 'DO innodb_redo_log_consumer_register("PXB");' query.
+
+* [PXB-3181]: Using `innodb-use-native-aio=true` printed duplicate timestamps.
+
+## Deprecated or removed
+
+
+
+## Useful links
+
+Install [Percona XtraBackup 8.2](..//installation.md)
+
+The [Percona XtraBackup GitHub repository](https://github.com/percona/percona-xtrabackup)
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
+
+[Percona XtraBackup 8.2]: https://docs.percona.com/percona-xtrabackup/inno-release/
+
+[PXB-3003]: https://jira.percona.com/browse/PXB-3003
```

Comment:
why are these not in the above bug fixes sections with a description?

Reply:
Just a different way to add Markdown links.